Libraries/CustomComponents/Navigator/Navigator.js

```diff
@@ -1095,7 +1101,7 @@ var Navigator = React.createClass({
     return (
       <View style={[styles.container, this.props.style]}>
         <View
-          style={styles.transitioner}
+          style={[styles.transitioner,{backgroundColor: this.props.transitionBackgroundColor}]}
```

Comment:
space after comma, and make sure your lines aren't longer than 80 chars.  I'd format this like so:

  style={[
    styles.transitioner,
    {backgroundColor: this.props.transitionBackgroundColor}
  ]}